src/components/Fingerprint.vue

```diff
@@ -119,13 +120,22 @@ export default class Fingerprint extends ( mixins( Messages ) as VueConstructor<
 			font-family: $font-family-sansserif;
 		}
 
+		$descriptionAliasIndent: 0.5em;
+		$fingerprintMinWidth: 260px;
+
 		&__label,
 		&__description,
 		&__aliases {
-			min-width: 260px;
+			min-width: $fingerprintMinWidth;
 			max-width: 420px;
 		}
 
+		&__description,
+		&__aliases {
+			min-width: calc(#{$fingerprintMinWidth} - #{$descriptionAliasIndent});
```

Comment:
What is about Opera mini ( https://caniuse.com/#feat=calc )

Reply:
Well spotted, I forgot about this one.
I'm trying to solve the following problem: limit the description/alias container width to 260px while also giving it a margin-left. `calc` is one way to do it, but we can't use that. I now added a wrapping element for description and alias to achieve this. Better ideas are welcome!